src/tiledb/cloud/bioimg/helpers.py

```diff
@@ -1,31 +1,32 @@
-import logging
+# import logging
 from typing import Sequence
 
 import tiledb
-from tiledb.cloud.utilities import get_logger
 
+# from tiledb.cloud.logging import get_logger
 
-def get_logger_wrapper(
-    verbose: bool = False,
-) -> logging.Logger:
-    """
-    Get a logger instance and log version information.
```

Comment:
@ktsitsi 👋 what do you think about replacing bioimg's custom logger with the new standard logger in tiledb.cloud.logging?